src/ui/common/src/components/pages/workflows/index.tsx

```diff
@@ -120,6 +120,25 @@ const WorkflowsPage: React.FC<Props> = ({ user, Layout = DefaultLayout }) => {
     return workflowTableRow;
   });
 
+  const sortColumns = [
+    {
+      name: 'Last Run',
+      sortAccessPath: ['last_run'],
+    },
+    {
+      name: 'Name',
+      sortAccessPath: ['name', 'name'],
+    },
+    {
+      name: 'Engine',
+      sortAccessPath: ['engine', 'engineName'],
+    },
+    {
+      name: 'Status',
+      sortAccessPath: ['name', 'status'],
```

Comment:
What is this sortAccessPath field? I assume that it allows users to sort by status, but order the results by the name of the item? Or does it alphabetize the statuses?

Reply:
It might not be the clearest name, but It's the sequence of keys to use within the data row object to get the field that should be used to sort. It's documented [here](https://github.com/aqueducthq/aqueduct/pull/846/files#diff-f4a84e68fbf99ea9144649623979796ffd0744dd8a42be4c80d8f2cccaa868faR47-R49).